cmake/Modules/ConfigureGoogleTest.cmake

```diff
@@ -14,6 +14,13 @@ elseif(CMAKE_CXX11_ABI)
     list(APPEND GTEST_CMAKE_ARGS " -DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=1")
 endif(NOT CMAKE_CXX11_ABI)
 
+# Poor's man workaround for
```

Comment:
Change to "workaround for..." (Avoids typo and gender/class insensitivity)

Reply:
See https://github.com/rapidsai/rmm/pull/547